### ISSUE TITLE
[Le Deep Learning en pratique avec Keras] Florient Chouteau

### ISSUE THREAD

**Issue by fchouteau** [TlseDataScience/meetup-cfp#10]
## Le speaker
Florient Chouteau, ingénieur Deep Learning chez Magellium | En régie à Airbus DS Intelligence

* Twitter handle:
@foxchouteau

## Sujet de votre conférence
**Le Deep Learning en pratique avec Keras**

## Description de votre conférence

Si l'on voulait mesurer l'effervescence du domaine du Deep Learning, nul doute que l'évolution des outils de développements figurerait parmis les métriques clés. C'est en effet un paysage en évolution constante, où la concurrence est rude et ouverte (dans tous les sens du terme), où les APIs peuvent être non-rétrocompatible du jour au lendemain, et où les choix techniques sont parfois difficiles. Que dire si l'on débute: l'étendue des possibilités peut s'avérer paralysante.  

Vers quelle librairie/framework/API s'orienter ? Cette conférence n'a pas la prétention de répondre à cette question, mais souhaite plutôt aborder l'une des possibiltiés, l'API Keras, dont la renommée n'a d'égale que l'ambition. 

A partir de l'ambition affichée (*Deep Learning for Humans*) par le créateur de l'API, on balaiera, au travers d'un même exemple et de snippets de code les plus concis possibles, plusieurs façon d'utiliser l'API. Notamment, pour:

- **Débuter** Comment Keras met en avant une API assez simple et compréhensible, en offrant des moyens à la fois assez "haut-niveau" et suffisamment clair pour que l'on puisse mettre directement les mains dans le cambouis,
- **Aller plus loin** Comment Keras donne le loisir à l'utilisateur d'appliquer des concepts, des méthodes plus avancées qu'il n'y paraît grâce à son extensibilité, et à quelques principes bien pensés,
- **Passer à l'échelle** Si l'objectif, au delà des tips and tricks and best practices du Deep Learning, est d'aller entraîner des modèles assez lourds sur des jeux de données importants, est-ce encore possible ? Cela sera notamment abordé au travers de tf.keras

Le tout dans une optique plus "expérimentale" que "théorique", ce qui semble convenir au mieux au Deep Learning

_Cette conférence sera abordée sous l'angle des réseaux de neurones convolutifs pour l'imagerie, dans un cadre "single-machine"_

## Informations diverses
* Thématique, Labels : Apprentissage Supervisé, Deep Learning, Frameworks Deep Learning, Image
* Niveau de difficulté (débutant|avancé|confirmé) : débutant|avancé
* Durée (quickie 15-30min, conférence >1h) : conférence > 1h
* Format (slides, live-coding, les deux): slides + notebooks à consulter avant/après si envie de pousser
* Dispo ou indispo (après mars, dès que possible, pas entre X et Y): 18 Juin 2018

**Comment by fchouteau** [comment]
Mise à jour de l'abstract pour le talk du 18
